internal/ethapi/api.go

```diff
@@ -1541,6 +1552,11 @@ func newRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNumber
 			result.ChainID = (*hexutil.Big)(id)
 		}
 
+		if tx.IsCeloLegacy() {
+			// If this is a celo legacy transaction then set eth compatible to false
+			result.EthCompatible = new(bool)
```

Comment:
Why not simply use `false`?

Reply:
Because EthCompatible is a pointer and I don't think there's a neat way to do it in one line, but I'd be happy to be proved wrong.